src/main/resources/templates/letter-to-prison_custody-office.html

```diff
@@ -267,18 +267,16 @@
   <div th:if="${hasAdditionalLicenceConditions}">
     <p class=MsoNormal style='margin-left:0cm;text-indent:-.1pt'>Additional Licence Conditions requested by Offender Manager - Please ensure these are included on the licence when the offender is re-released:</p>
 
-    <p class=MsoNormal style='margin-left:0cm;text-indent:-.1pt'><span th:text="${additionalLicenceConditionsDetail}"/></p>
+    <p class=MsoNormal style='margin-left:0cm;text-indent:-.1pt'><pre class=MsoNormal style='margin-left:0cm;text-indent:-.1pt;white-space: pre-wrap' th:text="${additionalLicenceConditionsDetail}"></pre></p>
```

Comment:
Do we need to reproduce the class setting from the para on the contained pre?  E.g. in this case that gives us 4 copies of `pre class="MsoNormal" style="white-space: pre-wrap`?

Reply:
It appeared we do (pre doesnt inherit styling)